samples/dotnet/FileCompression/FileCompression.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
uber nit: folder name of `file-compression-skills` or just `file-compression` would better follow the pattern we set in this directory. However, `KernelBuilder` and `KernelHttpServer` have both broken the patttern, hence the *uber* nit. Only do if it makes sense when rebasing/merging with main.

Reply:
Trying to get this one out quickly as people are waiting on me for SK Service stuff.  It might be worth to rename all "offending" folders in one go, though.  Leaving as is for now to move on to SK service.